### ISSUE TITLE
NuGet Runtime pack for community-supported architectures

### ISSUE THREAD

**Issue by driver1998** [dotnet/sdk#42248]
### Is your feature request related to a problem? Please describe.

.NET now has a few community-supported platforms: `linux-riscv64`, `linux-loongarch64`, etc. 

And to properly support cross-publishing in .NET SDK, we need to get runtime packages from NuGet, otherwise publish will fail.

Hosting these packages on self-hosted repos are certainly an option, but is it possible now for maintainers to push those to NuGet.org under their own account? (Not as Microsoft because they won't be supported by Microsoft to begin with).

That way we can get a consistent publish experience.

**Comment by marcpopMSFT** [comment]
Adding @richlander @MichaelSimons for a policy discussion around these other architectures.

**Comment by richlander** [comment]
This has been a problem for a long time. I think the gap here is a design.

> Hosting these packages on self-hosted repos are certainly an option, but is it possible now for maintainers to push those to NuGet.org under their own account? (Not as Microsoft because they won't be supported by Microsoft to begin with).

I think this description captures the problem pretty well.

@omajid @jkotas @baronfel

**Comment by omajid** [comment]
This also affects .NET on IBM platforms (eg, `linux-ppc64le`, `linux-s390x`)

cc @uweigand @Swapnali911

**Comment by omajid** [comment]
> is it possible now for maintainers to push those to NuGet.org under their own account?

AFAIK, the answer is currently no. The prefexes are reserved by Microsoft on nuget.org, and no one else can publish to the prefixes. Publishing under another prefix makes it non-trivial to consume, if it's possible at all.

**Comment by richlander** [comment]
Right. A design would need to enable publishing the various packs under arbitrary names and to provide a registration mechanism, likely via a NuGet package (or something equivalent).

**Comment by driver1998** [comment]
Workaround:
A guide (Chinese only) to manually extract runtime packs from native SDK to your build SDK for cross-publishing scenarios (like from windows-x64 to linux-loongarch64):

https://driver1998.github.io/posts/dotnet-cross-publish-loongarch/

**Comment by driver1998** [comment]
Note: 
If you have runtime pack nupkgs in hand (and a lot of downstream distributors do provide them now), you can build a local NuGet repo for easy cross-publishing.

The following nupkgs should be enough for most cases:
```
- Microsoft.AspNetCore.App.Runtime.{RID}.*.nupkg
- Microsoft.NETCore.App.Crossgen2.{RID}.*.nupkg
- Microsoft.NETCore.App.Host.{RID}.*.nupkg
- Microsoft.NETCore.App.Runtime.{RID}.*.nupkg
- runtime.{RID}.Microsoft.DotNet.ILCompiler.*.nupkg
```

Just put those in a directory and add that as a package source: 
```xml
<packageSources>
    <add key="LoongArch64 Packages" value="/data/loongarch64-nupkgs" />
</packageSources>
```
